05:45:28 PM:DEBUG [Logger_base.py:15] mensaje a nivel debug
05:45:28 PM:INFO [Logger_base.py:16] mensaje a nivel info
05:45:28 PM:WARNING [Logger_base.py:17] mensaje a nivel warning
05:45:28 PM:ERROR [Logger_base.py:18] mensaje a nivel error
05:45:28 PM:CRITICAL [Logger_base.py:19] mensaje a nivel critical
05:37:45 PM:DEBUG [Persona.py:52] 
            Id Persona: 1,
            Nombre: Juan,
            Apellido: Perez,
        
05:39:30 PM:DEBUG [Persona.py:52] 
            Id Persona: 1,
            Nombre: Juan,
            Apellido: Perez,
        
05:41:14 PM:DEBUG [Persona.py:52] 
            Id Persona: 1,
            Nombre: Juan,
            Apellido: Perez,
        
05:41:14 PM:DEBUG [Persona.py:54] 
            Id Persona: None,
            Nombre: Jose,
            Apellido: Lepez,
        
05:44:18 PM:DEBUG [Persona.py:53] 
            Id Persona: 1,
            Nombre: Juan,
            Apellido: Perez,
            Email: jperez@mail..com
        
05:44:18 PM:DEBUG [Persona.py:55] 
            Id Persona: None,
            Nombre: Jose,
            Apellido: Lepez,
            Email: ljose@mail.com
        
05:45:31 PM:DEBUG [Persona.py:53] 
            Id Persona: 1,
            Nombre: Juan,
            Apellido: Perez,
            Email: jperez@mail..com
        
05:45:31 PM:DEBUG [Persona.py:55] 
            Id Persona: None,
            Nombre: Jose,
            Apellido: Lepez,
            Email: ljose@mail.com
        
05:45:31 PM:DEBUG [Persona.py:57] 
            Id Persona: 1,
            Nombre: None,
            Apellido: None,
            Email: None
        
07:03:50 PM:DEBUG [conexion.py:23] Conexion  Exitosa: <connection object at 0x000002190F3BE370; dsn: 'user=postgres password=xxx dbname=test_bd host=127.0.0.1 port=5432', closed: 0>
07:03:50 PM:ERROR [conexion.py:40] Ocurrio un error: 'NoneType' object has no attribute 'cursor'
07:17:16 PM:ERROR [conexion.py:26] Ocurrio un error:connection to server at "127.0.0.1", port 5432 failed: FATAL:  password authentication failed for user "postgre"

07:17:40 PM:DEBUG [conexion.py:23] Conexion  Exitosa: <connection object at 0x000001BC04DAC370; dsn: 'user=postgres password=xxx dbname=test_bd host=127.0.0.1 port=5432', closed: 0>
07:18:28 PM:DEBUG [conexion.py:23] Conexion  Exitosa: <connection object at 0x000002013D9BC370; dsn: 'user=postgres password=xxx dbname=test_bd host=127.0.0.1 port=5432', closed: 0>
07:18:28 PM:ERROR [conexion.py:53] Ocurrio un error: 'NoneType' object has no attribute 'cursor'
07:21:13 PM:DEBUG [conexion.py:23] Conexion  Exitosa: <connection object at 0x0000016055BDC370; dsn: 'user=postgres password=xxx dbname=test_bd host=127.0.0.1 port=5432', closed: 0>
07:21:13 PM:ERROR [conexion.py:53] Ocurrio un error: 'NoneType' object has no attribute 'cursor'
07:22:05 PM:DEBUG [conexion.py:23] Conexion  Exitosa: <connection object at 0x000001F808DAC370; dsn: 'user=postgres password=xxx dbname=test_bd host=127.0.0.1 port=5432', closed: 0>
07:22:05 PM:DEBUG [conexion.py:50] Se abrio correctamente el cursor : <cursor object at 0x000001F808FE1820; closed: 0>
03:35:09 PM:DEBUG [conexion.py:23] Conexion  Exitosa: <connection object at 0x000001F75B2678C0; dsn: 'user=postgres password=xxx dbname=test_bd host=127.0.0.1 port=5432', closed: 0>
03:35:09 PM:DEBUG [conexion.py:50] Se abrio correctamente el cursor : <cursor object at 0x000001F75CF8DE40; closed: 0>
03:35:34 PM:DEBUG [conexion.py:23] Conexion  Exitosa: <connection object at 0x000001F0D61078C0; dsn: 'user=postgres password=xxx dbname=test_bd host=127.0.0.1 port=5432', closed: 0>
03:35:34 PM:DEBUG [conexion.py:50] Se abrio correctamente el cursor : <cursor object at 0x000001F0D7DE1F20; closed: 0>
03:35:34 PM:DEBUG [persona_dao.py:35] 
            Id Persona: 2,
            Nombre: Xarla ,
            Apellido: Gomez,
            Email: xgomez@mail.com
        
03:35:34 PM:DEBUG [persona_dao.py:35] 
            Id Persona: 6,
            Nombre: calos,
            Apellido: lara,
            Email: clara@mail.com
        
03:35:34 PM:DEBUG [persona_dao.py:35] 
            Id Persona: 7,
            Nombre: Marcos,
            Apellido: Canto,
            Email: mcanto@mail.com
        
03:35:34 PM:DEBUG [persona_dao.py:35] 
            Id Persona: 8,
            Nombre: Marcelo,
            Apellido: Cuenca,
            Email: ccanta@gmail.com
        
03:35:34 PM:DEBUG [persona_dao.py:35] 
            Id Persona: 9,
            Nombre: calos,
            Apellido: lara,
            Email: clara@mail.com
        
03:35:34 PM:DEBUG [persona_dao.py:35] 
            Id Persona: 10,
            Nombre: Marcos,
            Apellido: Canto,
            Email: mcanto@mail.com
        
03:35:34 PM:DEBUG [persona_dao.py:35] 
            Id Persona: 11,
            Nombre: Marcelo,
            Apellido: Cuenca,
            Email: ccanta@gmail.com
        
03:35:34 PM:DEBUG [persona_dao.py:35] 
            Id Persona: 12,
            Nombre: calos,
            Apellido: lara,
            Email: clara@mail.com
        
03:35:34 PM:DEBUG [persona_dao.py:35] 
            Id Persona: 13,
            Nombre: Marcos,
            Apellido: Canto,
            Email: mcanto@mail.com
        
03:35:34 PM:DEBUG [persona_dao.py:35] 
            Id Persona: 14,
            Nombre: Marcelo,
            Apellido: Cuenca,
            Email: ccanta@gmail.com
        
03:35:34 PM:DEBUG [persona_dao.py:35] 
            Id Persona: 16,
            Nombre: maria,
            Apellido: esparza,
            Email: mesparza@mail.com
        
03:35:34 PM:DEBUG [persona_dao.py:35] 
            Id Persona: 19,
            Nombre: carlos,
            Apellido: lara,
            Email: clara@mail.com
        
03:35:34 PM:DEBUG [persona_dao.py:35] 
            Id Persona: 20,
            Nombre: jorge,
            Apellido: prol,
            Email: jprol@mail.com
        
03:35:34 PM:DEBUG [persona_dao.py:35] 
            Id Persona: 21,
            Nombre: jorge,
            Apellido: prol,
            Email: jprol@mail.com
        
03:35:34 PM:DEBUG [persona_dao.py:35] 
            Id Persona: 22,
            Nombre: Alex,
            Apellido: rojas,
            Email: arojas@mail.com
        
03:46:58 PM:DEBUG [conexion.py:23] Conexion  Exitosa: <connection object at 0x00000286544678C0; dsn: 'user=postgres password=xxx dbname=test_bd host=127.0.0.1 port=5432', closed: 0>
03:46:58 PM:DEBUG [conexion.py:50] Se abrio correctamente el cursor : <cursor object at 0x0000028656172F20; closed: 0>
03:53:07 PM:DEBUG [conexion.py:23] Conexion  Exitosa: <connection object at 0x000001CB199B78C0; dsn: 'user=postgres password=xxx dbname=test_bd host=127.0.0.1 port=5432', closed: 0>
03:53:07 PM:DEBUG [conexion.py:50] Se abrio correctamente el cursor : <cursor object at 0x000001CB1B595E40; closed: 0>
03:54:31 PM:DEBUG [conexion.py:23] Conexion  Exitosa: <connection object at 0x000002A5F2AD78C0; dsn: 'user=postgres password=xxx dbname=test_bd host=127.0.0.1 port=5432', closed: 0>
03:54:31 PM:DEBUG [conexion.py:50] Se abrio correctamente el cursor : <cursor object at 0x000002A5F47D5E40; closed: 0>
03:57:09 PM:DEBUG [conexion.py:23] Conexion  Exitosa: <connection object at 0x0000026663CE78C0; dsn: 'user=postgres password=xxx dbname=test_bd host=127.0.0.1 port=5432', closed: 0>
03:57:09 PM:DEBUG [conexion.py:50] Se abrio correctamente el cursor : <cursor object at 0x00000266659F5E40; closed: 0>
04:16:02 PM:DEBUG [conexion.py:23] Conexion  Exitosa: <connection object at 0x000002AFC9C978C0; dsn: 'user=postgres password=xxx dbname=test_bd host=127.0.0.1 port=5432', closed: 0>
04:16:02 PM:DEBUG [conexion.py:50] Se abrio correctamente el cursor : <cursor object at 0x000002AFCB995E40; closed: 0>
04:17:09 PM:DEBUG [conexion.py:23] Conexion  Exitosa: <connection object at 0x000002263BF578C0; dsn: 'user=postgres password=xxx dbname=test_bd host=127.0.0.1 port=5432', closed: 0>
04:17:09 PM:DEBUG [conexion.py:50] Se abrio correctamente el cursor : <cursor object at 0x000002263DC75E40; closed: 0>
04:17:50 PM:DEBUG [conexion.py:23] Conexion  Exitosa: <connection object at 0x00000210EA2C78C0; dsn: 'user=postgres password=xxx dbname=test_bd host=127.0.0.1 port=5432', closed: 0>
04:17:50 PM:DEBUG [conexion.py:50] Se abrio correctamente el cursor : <cursor object at 0x00000210EBFB5E40; closed: 0>
04:19:13 PM:DEBUG [conexion.py:23] Conexion  Exitosa: <connection object at 0x000001DF984378C0; dsn: 'user=postgres password=xxx dbname=test_bd host=127.0.0.1 port=5432', closed: 0>
04:19:13 PM:DEBUG [conexion.py:50] Se abrio correctamente el cursor : <cursor object at 0x000001DF9A144E40; closed: 0>
04:25:03 PM:DEBUG [conexion.py:23] Conexion  Exitosa: <connection object at 0x00000212286C78C0; dsn: 'user=postgres password=xxx dbname=test_bd host=127.0.0.1 port=5432', closed: 0>
04:25:03 PM:DEBUG [conexion.py:50] Se abrio correctamente el cursor : <cursor object at 0x000002122A3AEE40; closed: 0>
04:25:16 PM:DEBUG [conexion.py:23] Conexion  Exitosa: <connection object at 0x000001B1D66D78C0; dsn: 'user=postgres password=xxx dbname=test_bd host=127.0.0.1 port=5432', closed: 0>
04:25:16 PM:DEBUG [conexion.py:50] Se abrio correctamente el cursor : <cursor object at 0x000001B1D82CEE40; closed: 0>
04:32:28 PM:DEBUG [conexion.py:23] Conexion  Exitosa: <connection object at 0x000001DAFFB978C0; dsn: 'user=postgres password=xxx dbname=test_bd host=127.0.0.1 port=5432', closed: 0>
04:32:28 PM:DEBUG [conexion.py:50] Se abrio correctamente el cursor : <cursor object at 0x000001DA81895E40; closed: 0>
04:38:27 PM:DEBUG [conexion.py:23] Conexion  Exitosa: <connection object at 0x0000016E2F2D78C0; dsn: 'user=postgres password=xxx dbname=test_bd host=127.0.0.1 port=5432', closed: 0>
04:38:27 PM:DEBUG [conexion.py:50] Se abrio correctamente el cursor : <cursor object at 0x0000016E30FE6E40; closed: 0>
04:41:21 PM:DEBUG [conexion.py:23] Conexion  Exitosa: <connection object at 0x000002D5F0AD78C0; dsn: 'user=postgres password=xxx dbname=test_bd host=127.0.0.1 port=5432', closed: 0>
04:41:21 PM:DEBUG [conexion.py:50] Se abrio correctamente el cursor : <cursor object at 0x000002D5F27B3F20; closed: 0>
04:43:59 PM:DEBUG [conexion.py:23] Conexion  Exitosa: <connection object at 0x000001F919B278C0; dsn: 'user=postgres password=xxx dbname=test_bd host=127.0.0.1 port=5432', closed: 0>
04:43:59 PM:DEBUG [conexion.py:50] Se abrio correctamente el cursor : <cursor object at 0x000001F91B833F20; closed: 0>
04:43:59 PM:DEBUG [persona_dao.py:80] 
            Id Persona: 2,
            Nombre: Xarla ,
            Apellido: Gomez,
            Email: xgomez@mail.com
        
04:43:59 PM:DEBUG [persona_dao.py:80] 
            Id Persona: 6,
            Nombre: calos,
            Apellido: lara,
            Email: clara@mail.com
        
04:43:59 PM:DEBUG [persona_dao.py:80] 
            Id Persona: 7,
            Nombre: Marcos,
            Apellido: Canto,
            Email: mcanto@mail.com
        
04:43:59 PM:DEBUG [persona_dao.py:80] 
            Id Persona: 8,
            Nombre: Marcelo,
            Apellido: Cuenca,
            Email: ccanta@gmail.com
        
04:43:59 PM:DEBUG [persona_dao.py:80] 
            Id Persona: 9,
            Nombre: calos,
            Apellido: lara,
            Email: clara@mail.com
        
04:43:59 PM:DEBUG [persona_dao.py:80] 
            Id Persona: 10,
            Nombre: Marcos,
            Apellido: Canto,
            Email: mcanto@mail.com
        
04:43:59 PM:DEBUG [persona_dao.py:80] 
            Id Persona: 11,
            Nombre: Marcelo,
            Apellido: Cuenca,
            Email: ccanta@gmail.com
        
04:43:59 PM:DEBUG [persona_dao.py:80] 
            Id Persona: 12,
            Nombre: calos,
            Apellido: lara,
            Email: clara@mail.com
        
04:43:59 PM:DEBUG [persona_dao.py:80] 
            Id Persona: 13,
            Nombre: Marcos,
            Apellido: Canto,
            Email: mcanto@mail.com
        
04:43:59 PM:DEBUG [persona_dao.py:80] 
            Id Persona: 14,
            Nombre: Marcelo,
            Apellido: Cuenca,
            Email: ccanta@gmail.com
        
04:43:59 PM:DEBUG [persona_dao.py:80] 
            Id Persona: 16,
            Nombre: maria,
            Apellido: esparza,
            Email: mesparza@mail.com
        
04:43:59 PM:DEBUG [persona_dao.py:80] 
            Id Persona: 19,
            Nombre: carlos,
            Apellido: lara,
            Email: clara@mail.com
        
04:43:59 PM:DEBUG [persona_dao.py:80] 
            Id Persona: 20,
            Nombre: jorge,
            Apellido: prol,
            Email: jprol@mail.com
        
04:43:59 PM:DEBUG [persona_dao.py:80] 
            Id Persona: 21,
            Nombre: jorge,
            Apellido: prol,
            Email: jprol@mail.com
        
04:43:59 PM:DEBUG [persona_dao.py:80] 
            Id Persona: 22,
            Nombre: Alex,
            Apellido: rojas,
            Email: arojas@mail.com
        
